.github/workflows/pr-validation.yaml

```diff
@@ -2,7 +2,7 @@ name: Pull Request Validation
 on:
   pull_request:
     types: [opened, synchronize, reopened, edited]
-    branches: [master]
+    branches: [main]
```

Comment:
Nit: You can keep both `master` and `main` here for a transitional period.

Reply:
I'm not sure what happen with GH Actions trying to execute a workflow for a nonexistent branch, do you know?

Reply:
It depends on the situation. In this case, the branch name does not make a difference since it will just use a different branch. In many cases it will just error out.

Reply:
I got it. But anyway I think we don't it. If something goes really wrong we can just revert the whole thing (revert this PR and rename back the branch to `master`)